frontend/src/pages/destructionlist/hooks.ts

```diff
@@ -289,10 +289,31 @@ export function getFields(
       type: "string",
       filterLookup: "toelichting__icontains",
     },
-    // TODO
-    // {
-    //   name: "Behandelend afdeling"
-    // },
+    {
+      name: "Behandelend afdeling",
+      active: false,
+      type: "string",
+      filterLookup: "behandelend_afdeling",
```

Comment:
Please tell me if I'm nitpicking here. But wouldn't it be more consistent with other filter (params) if the param would be: "behandelend_afdeling__icontains"

Reply:
No, makes sense! Will change